README.md

```diff
@@ -39,54 +41,105 @@ cmake ..
 make
 ```
 
-### Development
+Or you can simply use `catkin_make` instead.
+```sh
+cd catkin_ws
+catkin_make 
```

Comment:
Can we `catkin_make`? Or do we still need to run `catkin_make_isolated`

Reply:
For the Hivemind we can use catking make. isolated is only needed for swarmus-ros